usaspending_api/submissions/management/commands/rm_submission.py

```diff
@@ -46,10 +44,22 @@ def signal_handler(signal, frame):
 
         deleted_stats = submission.delete()
 
-        self.logger.info("Finished deletions.")
+        models = {
```

Comment:
This is a really nice addition!

Reply:
Glad my own discomfort with the logging was felt by others